src/itemdb.d

```diff
@@ -484,4 +484,18 @@ final class ItemDatabase
 		auto stmt = db.prepare("VACUUM;");
 		stmt.exec();
 	}
+	
+	// Select distinct driveId items from database
+	string[] selectDistinctDriveIds()
+	{
+		string[] driveIdArray;
+		auto stmt = db.prepare("SELECT DISTINCT driveId FROM item;");
+		auto res = stmt.exec();
```

Comment:
Maybe we need to protect that against broken sql database and assertions, similar to another recent PR? (resync necessary)

Reply:
Not required here, as we are not building any complex path or complex additional SQL query based on the prior response.

`SELECT DISTINCT driveId FROM item;` will return zero rows or rows of distinct driveId's. If no rows are returned, an empty array is returned to the calling function, which has then no driveid's to iterate through, thus, path is not in sync in DB.

Reply:
Ok, thank!